### PR TITLE
Update CIBC

### PR DESCRIPTION
**Pull request by sarahmrrp**
Add information about CIBC app 2FA
Remove email 2FA as no longer supported

### CODE REVIEW

File: entries/c/cibc.com.json [hunk 1]
```diff
@@ -5,14 +5,17 @@
...
-      "email"
+      "custom-software"
     ],
-    "documentation": "https://www.cibc.com/en/privacy-security/two-step-verification-b.html",
-    "regions": [
-      "ca"
+    "custom-software": [
+      "CIBC App Push Notification"
     ],
+    "documentation": "https://www.cibc.com/en/privacy-security/two-step-verification-b.html",
```
Comment (Carlgo11) [inline entries/c/cibc.com.json:13]: This page this references email as a authentication method.

**Comment by Carlgo11** [comment]
As the requested changes haven't been resolved yet I'm going to label this PR as inactive.
:warning: Unless there's activity within the next 7 days I'll close the PR.

**Comment by Carlgo11** [comment]
Closing as #6993 was merged.

File: entries/c/cibc.com.json (hunk 1 above)
Comment (jpayoung) [reply entries/c/cibc.com.json:13]: Indeed! The page referenced here is out-of-date. This change would add the current one:

```suggestion
    "documentation": "https://www.cibc.com/en/privacy-security/two-step-verification.html",
```

**Comment by jpayoung** [comment]
Hey @Carlgo11! 

Looks like both #6993 and this pull request reference an old page.
This is the old page: https://www.cibc.com/en/privacy-security/two-step-verification-b.html (currently referenced)
This is the new page: https://www.cibc.com/en/privacy-security/two-step-verification.html

The new page states that CIBC will no longer send email verification codes:

CIBC's webpage with highlighted text stating they no longer send one-time verification codes via email.

It is a very non-obvious distinguishment between the new page and the old page. I don't know why CIBC hasn't removed the old page yet.

**Comment by jpayoung** [comment]
To implement this change, should it be done here or should I file a new pull request? (I don't want to step on @sarahgguwu's toes.)

Thanks so much for all your work. Let me know how I can help. Hope I'm not being too annoying! 😉

**Pull request closed without merging**